src/components/OrganizationsDropdown.tsx

```diff
@@ -69,18 +43,13 @@ export default function OrganizationsDropdown(): JSX.Element {
   return (
     <div>
       <AddNewOrganizationModal open={newOrganizationModalOpened} onCancel={onCloseCreateOrganizationModal} />
-      <IconButton onClick={handleClick} size='small' className={classes.iconContainer}>
-        <p>{selectedOrganization.name}</p>
-        <Icon name='chevronDown' size='medium' className={classes.icon} />
-      </IconButton>
       <PopoverMenu
-        sections={[
+        anchor={<p>{selectedOrganization.name}</p>}
```

Comment:
I'm seeing the name a little bigger now, I think the iconContainer class was giving the fontSize to this <p>

Reply:
ok i will add a class name for this

Reply:
the other changes seem ok, they all have explicit font sizes.. good catch